OnItemEnter增加队长/团长职业、种族显示

hitechbeijing opened the pull request:
既然有LeaderClass方法我就用

hitechbeijing committed:
OnItemEnter增加队长/团长职业显示

hitechbeijing committed:
修复缺少一个职业翻译问题

hitechbeijing committed:
改进方法实现职业显示

hitechbeijing committed:
增加队长/团长种族显示

hitechbeijing committed:
职业颜色区分

hitechbeijing committed:
主界面增加一列职业，显示更直观

hitechbeijing committed:
增加闲置超时开关，重启游戏生效

jki14 commented:
Thanks for sharing the improvement.  
Sorry for the late review.  
By the way, I'd like to reiterate that I am not the original author of MeetingHorn. All the comments only based on my individual ideas and thoughts.

jki14 commented on MeetingHorn/Core/LFG.lua:
```diff
@@ -59,7 +58,11 @@ function LFG:OnEnable()
         return self:OnIdleTimer()
     end)
-    self.idleTimer:Start(5)
+    -- print(ns.Addon.db.profile.options['idleTimer'])
```
Please consider dropping the hacked debug message printing rather than commenting out the code.

jki14 commented on MeetingHorn/Core/LFG.lua:
```diff
@@ -114,7 +117,7 @@ function LFG:OnEnable()
     end)
 
-    C_Timer.After(5, function()
+    C_Timer.After(10, function()
```
Would you mind to share any strong reason to change this default hardcoded delay?  
Otherwise, I slightly prefer to keep it or create an option to configure it rather than keep changing the hardcoded delay.

jki14 commented on MeetingHorn/Core/LFG.lua:
```diff
@@ -647,4 +650,4 @@ function LFG:IsFilter(text)
         return self.filters[text]
     end
-end
+end
```
[Minor] Please consider fixing the file format issue.

jki14 commented on MeetingHorn/Localization/zhCN.lua:
```diff
@@ -9,6 +9,7 @@ L.ADDON_NAME = '集结号'
 L['Leader'] = '队长'
 L['Comment'] = '说明'
 L['Operation'] = '操作'
+L['Class'] = '职业'
```
It seems that this change might not belong to this commit.

hitechbeijing replied:
I have add the class both in OnItemEnter and main window.

jki14 commented on MeetingHorn/Localization/zhCN.lua:
```diff
@@ -107,6 +108,7 @@ L['删除成功，关键字“%s”已删除。'] = true
 
 L['Show data broker'] = '显示悬浮窗'
 L['Hide activity in chat frame'] = '隐藏聊天窗口内的集结号活动'
+L['Idle Timer'] = '开启/关闭闲置超时(reload后生效)'
```
It might be possible to switch the Idle Timer without reloading.

hitechbeijing replied:
The local database need to refresh, so can we use the refresh time to replace Idle time?

hitechbeijing replied:
Some players said they want refresh data manual, so I add the switch.

jki14 commented on MeetingHorn/UI/Options.lua:
```diff
@@ -46,6 +46,7 @@ function Options:Constructor()
             databroker = toggle(L['Show data broker']),
             chatfilter = toggle(L['Hide activity in chat frame']),
             activityfilter = toggle(L['启用关键字过滤']),
+            idleTimer = toggle(L['Idle Timer']),
```
[Style] I prefer to use `idletimer` instead of `idleTimer` for keeping the coding style consistent.

jki14 commented on MeetingHorn/Core/Activity.lua:
```diff
@@ -120,7 +121,9 @@ end
+function Activity:GetLeaderLocClass()
+    return self.leaderLocClass
+end
 function Activity:GetActivityId()
```
[Style] Please consider leaving blank lines between functions for keeping the coding style consistent.

jki14 commented on MeetingHorn/Localization/zhCN.lua:
```diff
@@ -24,15 +24,15 @@ L['Update Activity'] = '更新活动'
+-- L['WARLOCK'] = '术士'
+-- L['DRUID'] = '德鲁伊'
+-- L['ROGUE'] = '盗贼'
+-- L['PALADIN'] = '圣骑士'
```
Please consider dropping these lines rather than comment out the codes.

jki14 commented on MeetingHorn/Core/Activity.lua:
```diff
@@ -239,6 +242,7 @@ end
 function Activity:SetLeaderGUID(guid)
     self.guid = guid
     self.leaderClass = guid and select(2, GetPlayerInfoByGUID(guid)) or 'PRIEST'
+    self.leaderLocClass = guid and select(1, GetPlayerInfoByGUID(guid)) or 'PRIEST'
```
It might be not necessary to keep both englishClass and localizedClass.

hitechbeijing replied:
it use for display localized Class, or it just display English Class. The second way is use the loc file to translate.

jki14 commented on MeetingHorn/UI/Browser.lua:
```diff
@@ -150,9 +150,10 @@ function Browser:Constructor()
+        local Class = item:GetLeaderLocClass()
+        if Class then
+            GameTooltip:AddLine(format('%s |cff%02x%02x%02x%s|r', CLASS, r * 255, g * 255, b * 255,
+            item:GetLeaderLocClass()), 1, 1, 1)
```
Please consider making the long line wrapping easier to read. For example:
```lua
GameTooltip:AddLine(
        format('%s |cff%02x%02x%02x%s|r', CLASS, r * 255, g * 255, b * 255,
               item:GetLeaderLocClass()),
        1, 1, 1)
```

hitechbeijing replied:
sure

jki14 commented on MeetingHorn/UI/Browser.lua:
```diff
@@ -23,14 +23,15 @@ function Browser:Constructor()
+    -- self.Header3:Disable()
+    -- self.Header4:Disable()
+    -- self.Header5:Disable()
+    -- self.Header6:Disable()
```
Would you mind sharing the reason why you prefer to enable the headers?

jki14 commented on MeetingHorn/UI/Browser.lua:
```diff
@@ -108,7 +111,6 @@ function Browser:Constructor()
         else
             button.Members:SetText('-')
         end
-
```
It seems like an accident blank line dropping.

jki14 reviewed (commented):
Thanks for your kind sharing about your private improvements about the MeetingHorn.  
I have made some production-standard code reviews on your commits, which works for discussion about how we could raise the code quality bar of the MeetingHorn.  
Please feel free to ignore them if it annoyed you. And please enjoy the great game rather than taking too much time to follow-up on the code reviews.

hitechbeijing commented:
So, can you understand Chinese or switch English next time?

发自我的iPhone

> 在 2020年3月2日，22:05，jki14 <notifications@github.com> 写道：
> 
> ﻿
> @jki14 commented on this pull request.
> 
> Thanks for your kind sharing about your private improvements about the MeetingHorn.
> I have made some production-standard code reviews on your commits, which works for discussion about how we could raise the code quality bar of the MeetingHorn.
> Please feel free to ignore them if it annoyed you. And please enjoy the great game rather than taking too much time to follow-up on the code reviews.
> 
> In MeetingHorn/Core/LFG.lua:
> 
> > @@ -59,7 +58,11 @@ function LFG:OnEnable()
>      self.idleTimer = ns.Timer:New(function()
>          return self:OnIdleTimer()
>      end)
> -    self.idleTimer:Start(5)
> +    -- print(ns.Addon.db.profile.options['idleTimer'])
> Please consider dropping the hacked debug message printing rather than commenting out the code.
> 
> In MeetingHorn/Core/LFG.lua:
> 
> > @@ -114,7 +117,7 @@ function LFG:OnEnable()
>          self:ZONE_CHANGED_NEW_AREA()
>      end)
>  
> -    C_Timer.After(5, function()
> +    C_Timer.After(10, function()
> Would you mind to share any strong reason to change this default hardcoded delay?
> Otherwise, I slightly prefer to keep it or create an option to configure it rather than keep changing the hardcoded delay.
> 
> In MeetingHorn/Core/LFG.lua:
> 
> > @@ -647,4 +650,4 @@ function LFG:IsFilter(text)
>      if text then
>          return self.filters[text]
>      end
> -end
> +end
> [Minor] Please consider fixing the file format issue.
> 
> In MeetingHorn/Localization/zhCN.lua:
> 
> > @@ -9,6 +9,7 @@ L.ADDON_NAME = '集结号'
>  L['Leader'] = '队长'
>  L['Comment'] = '说明'
>  L['Operation'] = '操作'
> +L['Class'] = '职业'
> It seems that this change might not belong to this commit.
> 
> In MeetingHorn/Localization/zhCN.lua:
> 
> > @@ -107,6 +108,7 @@ L['删除成功，关键字“%s”已删除。'] = true
>  
>  L['Show data broker'] = '显示悬浮窗'
>  L['Hide activity in chat frame'] = '隐藏聊天窗口内的集结号活动'
> +L['Idle Timer'] = '开启/关闭闲置超时(reload后生效)'
> It might be possible to switch the Idle Timer without reloading.
> 
> In MeetingHorn/UI/Options.lua:
> 
> > @@ -46,6 +46,7 @@ function Options:Constructor()
>              databroker = toggle(L['Show data broker']),
>              chatfilter = toggle(L['Hide activity in chat frame']),
>              activityfilter = toggle(L['启用关键字过滤']),
> +            idleTimer = toggle(L['Idle Timer']),
> [Style] I prefer to use idletimer instead of idleTimer for keeping the coding style consistent.
> 
> In MeetingHorn/Core/Activity.lua:
> 
> > @@ -120,7 +121,9 @@ end
>  function Activity:GetLeaderClass()
>      return self.leaderClass
>  end
> -
> +function Activity:GetLeaderLocClass()
> +    return self.leaderLocClass
> +end
>  function Activity:GetActivityId()
> [Style] Please consider leaving blank lines between functions for keeping the coding style consistent.
> 
> In MeetingHorn/Localization/zhCN.lua:
> 
> > +-- L['MAGE'] = '法师'
> +-- L['HUNTER'] = '猎人'
> +-- L['WARRIOR'] = '战士'
> +-- L['PRIEST'] = '牧师'
> +-- L['SHAMAN'] = '萨满祭司'
> +-- L['WARLOCK'] = '术士'
> +-- L['DRUID'] = '德鲁伊'
> +-- L['ROGUE'] = '盗贼'
> +-- L['PALADIN'] = '圣骑士'
> Please consider dropping these lines rather than comment out the codes.
> 
> In MeetingHorn/Core/Activity.lua:
> 
> >      self.leaderClass = guid and select(2, GetPlayerInfoByGUID(guid)) or 'PRIEST'
> +    self.leaderLocClass = guid and select(1, GetPlayerInfoByGUID(guid)) or 'PRIEST'
> It might be not necessary to keep both englishClass and localizedClass.
> 
> In MeetingHorn/UI/Browser.lua:
> 
> > +            GameTooltip:AddLine(format('%s |cff%02x%02x%02x%s|r', CLASS, r * 255, g * 255, b * 255,
> +            item:GetLeaderLocClass()), 1, 1, 1)
> Please consider making the long line wrapping easier to read. For example:
> 
> GameTooltip:AddLine(
>         format('%s |cff%02x%02x%02x%s|r', CLASS, r * 255, g * 255, b * 255,
>                item:GetLeaderLocClass()),
>         1, 1, 1)
> In MeetingHorn/UI/Browser.lua:
> 
> >      -- self.Header1:Disable()
> -    self.Header2:Disable()
> -    self.Header3:Disable()
> -    self.Header4:Disable()
> -    self.Header5:Disable()
> -    self.Header6:Disable()
> +    -- self.Header2:Disable()
> +    -- self.Header3:Disable()
> +    -- self.Header4:Disable()
> +    -- self.Header5:Disable()
> +    -- self.Header6:Disable()
> Would you mind sharing the reason why you prefer to enable the headers?
> 
> In MeetingHorn/UI/Browser.lua:
> 
> > @@ -108,7 +111,6 @@ function Browser:Constructor()
>          else
>              button.Members:SetText('-')
>          end
> -
> It seems like an accident blank line dropping.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

jki14 commented:
> So, can you understand Chinese or switch English next time? 发自我的iPhone
> […](#)
> 在 2020年3月2日，22:05，jki14 ***@***.***> 写道： ﻿ @jki14 commented on this pull request. Thanks for your kind sharing about your private improvements about the MeetingHorn. I have made some production-standard code reviews on your commits, which works for discussion about how we could raise the code quality bar of the MeetingHorn. Please feel free to ignore them if it annoyed you. And please enjoy the great game rather than taking too much time to follow-up on the code reviews. In MeetingHorn/Core/LFG.lua: > @@ -59,7 +58,11 @@ function LFG:OnEnable() self.idleTimer = ns.Timer:New(function() return self:OnIdleTimer() end) - self.idleTimer:Start(5) + -- print(ns.Addon.db.profile.options['idleTimer']) Please consider dropping the hacked debug message printing rather than commenting out the code. In MeetingHorn/Core/LFG.lua: > @@ -114,7 +117,7 @@ function LFG:OnEnable() self:ZONE_CHANGED_NEW_AREA() end) - C_Timer.After(5, function() + C_Timer.After(10, function() Would you mind to share any strong reason to change this default hardcoded delay? Otherwise, I slightly prefer to keep it or create an option to configure it rather than keep changing the hardcoded delay. In MeetingHorn/Core/LFG.lua: > @@ -647,4 +650,4 @@ function LFG:IsFilter(text) if text then return self.filters[text] end -end +end [Minor] Please consider fixing the file format issue. In MeetingHorn/Localization/zhCN.lua: > @@ -9,6 +9,7 @@ L.ADDON_NAME = '集结号' L['Leader'] = '队长' L['Comment'] = '说明' L['Operation'] = '操作' +L['Class'] = '职业' It seems that this change might not belong to this commit. In MeetingHorn/Localization/zhCN.lua: > @@ -107,6 +108,7 @@ L['删除成功，关键字“%s”已删除。'] = true L['Show data broker'] = '显示悬浮窗' L['Hide activity in chat frame'] = '隐藏聊天窗口内的集结号活动' +L['Idle Timer'] = '开启/关闭闲置超时(reload后生效)' It might be possible to switch the Idle Timer without reloading. In MeetingHorn/UI/Options.lua: > @@ -46,6 +46,7 @@ function Options:Constructor() databroker = toggle(L['Show data broker']), chatfilter = toggle(L['Hide activity in chat frame']), activityfilter = toggle(L['启用关键字过滤']), + idleTimer = toggle(L['Idle Timer']), [Style] I prefer to use idletimer instead of idleTimer for keeping the coding style consistent. In MeetingHorn/Core/Activity.lua: > @@ -120,7 +121,9 @@ end function Activity:GetLeaderClass() return self.leaderClass end - +function Activity:GetLeaderLocClass() + return self.leaderLocClass +end function Activity:GetActivityId() [Style] Please consider leaving blank lines between functions for keeping the coding style consistent. In MeetingHorn/Localization/zhCN.lua: > +-- L['MAGE'] = '法师' +-- L['HUNTER'] = '猎人' +-- L['WARRIOR'] = '战士' +-- L['PRIEST'] = '牧师' +-- L['SHAMAN'] = '萨满祭司' +-- L['WARLOCK'] = '术士' +-- L['DRUID'] = '德鲁伊' +-- L['ROGUE'] = '盗贼' +-- L['PALADIN'] = '圣骑士' Please consider dropping these lines rather than comment out the codes. In MeetingHorn/Core/Activity.lua: > self.leaderClass = guid and select(2, GetPlayerInfoByGUID(guid)) or 'PRIEST' + self.leaderLocClass = guid and select(1, GetPlayerInfoByGUID(guid)) or 'PRIEST' It might be not necessary to keep both englishClass and localizedClass. In MeetingHorn/UI/Browser.lua: > + GameTooltip:AddLine(format('%s |cff%02x%02x%02x%s|r', CLASS, r * 255, g * 255, b * 255, + item:GetLeaderLocClass()), 1, 1, 1) Please consider making the long line wrapping easier to read. For example: GameTooltip:AddLine( format('%s |cff%02x%02x%02x%s|r', CLASS, r * 255, g * 255, b * 255, item:GetLeaderLocClass()), 1, 1, 1) In MeetingHorn/UI/Browser.lua: > -- self.Header1:Disable() - self.Header2:Disable() - self.Header3:Disable() - self.Header4:Disable() - self.Header5:Disable() - self.Header6:Disable() + -- self.Header2:Disable() + -- self.Header3:Disable() + -- self.Header4:Disable() + -- self.Header5:Disable() + -- self.Header6:Disable() Would you mind sharing the reason why you prefer to enable the headers? In MeetingHorn/UI/Browser.lua: > @@ -108,7 +111,6 @@ function Browser:Constructor() else button.Members:SetText('-') end - It seems like an accident blank line dropping. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

使用中文是没有问题的, 因为之前的commit message都是以英文编写所以我的comment也使用了英文. 如带来任何不便请谅解.

hitechbeijing committed:
Merge remote-tracking branch 'upstream/master'

hitechbeijing commented:
没事，我只是看到项目描述用中文，我也写中文了。下次commit用双语。

发自我的iPhone

> 在 2020年3月3日，07:25，jki14 <notifications@github.com> 写道：
> 
> ﻿
> So, can you understand Chinese or switch English next time? 发自我的iPhone
> …
> 在 2020年3月2日，22:05，jki14 @.***> 写道： ﻿ @jki14 commented on this pull request. Thanks for your kind sharing about your private improvements about the MeetingHorn. I have made some production-standard code reviews on your commits, which works for discussion about how we could raise the code quality bar of the MeetingHorn. Please feel free to ignore them if it annoyed you. And please enjoy the great game rather than taking too much time to follow-up on the code reviews. In MeetingHorn/Core/LFG.lua: > @@ -59,7 +58,11 @@ function LFG:OnEnable() self.idleTimer = ns.Timer:New(function() return self:OnIdleTimer() end) - self.idleTimer:Start(5) + -- print(ns.Addon.db.profile.options['idleTimer']) Please consider dropping the hacked debug message printing rather than commenting out the code. In MeetingHorn/Core/LFG.lua: > @@ -114,7 +117,7 @@ function LFG:OnEnable() self:ZONE_CHANGED_NEW_AREA() end) - C_Timer.After(5, function() + C_Timer.After(10, function() Would you mind to share any strong reason to change this default hardcoded delay? Otherwise, I slightly prefer to keep it or create an option to configure it rather than keep changing the hardcoded delay. In MeetingHorn/Core/LFG.lua: > @@ -647,4 +650,4 @@ function LFG:IsFilter(text) if text then return self.filters[text] end -end +end [Minor] Please consider fixing the file format issue. In MeetingHorn/Localization/zhCN.lua: > @@ -9,6 +9,7 @@ L.ADDON_NAME = '集结号' L['Leader'] = '队长' L['Comment'] = '说明' L['Operation'] = '操作' +L['Class'] = '职业' It seems that this change might not belong to this commit. In MeetingHorn/Localization/zhCN.lua: > @@ -107,6 +108,7 @@ L['删除成功，关键字“%s”已删除。'] = true L['Show data broker'] = '显示悬浮窗' L['Hide activity in chat frame'] = '隐藏聊天窗口内的集结号活动' +L['Idle Timer'] = '开启/关闭闲置超时(reload后生效)' It might be possible to switch the Idle Timer without reloading. In MeetingHorn/UI/Options.lua: > @@ -46,6 +46,7 @@ function Options:Constructor() databroker = toggle(L['Show data broker']), chatfilter = toggle(L['Hide activity in chat frame']), activityfilter = toggle(L['启用关键字过滤']), + idleTimer = toggle(L['Idle Timer']), [Style] I prefer to use idletimer instead of idleTimer for keeping the coding style consistent. In MeetingHorn/Core/Activity.lua: > @@ -120,7 +121,9 @@ end function Activity:GetLeaderClass() return self.leaderClass end - +function Activity:GetLeaderLocClass() + return self.leaderLocClass +end function Activity:GetActivityId() [Style] Please consider leaving blank lines between functions for keeping the coding style consistent. In MeetingHorn/Localization/zhCN.lua: > +-- L['MAGE'] = '法师' +-- L['HUNTER'] = '猎人' +-- L['WARRIOR'] = '战士' +-- L['PRIEST'] = '牧师' +-- L['SHAMAN'] = '萨满祭司' +-- L['WARLOCK'] = '术士' +-- L['DRUID'] = '德鲁伊' +-- L['ROGUE'] = '盗贼' +-- L['PALADIN'] = '圣骑士' Please consider dropping these lines rather than comment out the codes. In MeetingHorn/Core/Activity.lua: > self.leaderClass = guid and select(2, GetPlayerInfoByGUID(guid)) or 'PRIEST' + self.leaderLocClass = guid and select(1, GetPlayerInfoByGUID(guid)) or 'PRIEST' It might be not necessary to keep both englishClass and localizedClass. In MeetingHorn/UI/Browser.lua: > + GameTooltip:AddLine(format('%s |cff%02x%02x%02x%s|r', CLASS, r * 255, g * 255, b * 255, + item:GetLeaderLocClass()), 1, 1, 1) Please consider making the long line wrapping easier to read. For example: GameTooltip:AddLine( format('%s |cff%02x%02x%02x%s|r', CLASS, r * 255, g * 255, b * 255, item:GetLeaderLocClass()), 1, 1, 1) In MeetingHorn/UI/Browser.lua: > -- self.Header1:Disable() - self.Header2:Disable() - self.Header3:Disable() - self.Header4:Disable() - self.Header5:Disable() - self.Header6:Disable() + -- self.Header2:Disable() + -- self.Header3:Disable() + -- self.Header4:Disable() + -- self.Header5:Disable() + -- self.Header6:Disable() Would you mind sharing the reason why you prefer to enable the headers? In MeetingHorn/UI/Browser.lua: > @@ -108,7 +111,6 @@ function Browser:Constructor() else button.Members:SetText('-') end - It seems like an accident blank line dropping. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.
> 
> 使用中文是没有问题的, 因为之前的commit message都是以英文编写所以我的comment也使用了英文. 如带来任何不便请谅解.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

hitechbeijing committed:
有NGA玩家反映一些地下城的等级限制不太合理，这里修复一下
Some NGA players report that some dungeon level restrictions are not reasonable, here is a fx

hitechbeijing committed:
Some NGA players report that some dungeon level restrictions are not reasonable, fix it here
有NGA玩家反映一些地下城的等级限制不太合理,修复一下

hitechbeijing committed:
删除一些注释

hitechbeijing committed:
删除部分注释

hitechbeijing committed:
keeping the coding style consistent

hitechbeijing committed:
fix syntax issue

hitechbeijing committed:
the reset button could clean search result noww
重设按钮可以清空搜索结果（即重新搜索）

hitechbeijing committed:
改进reset按钮刷新机制

hitechbeijing committed:
创建活动中斯坦索姆和黑石塔等级限制调整(以旅店老板为准)

hitechbeijing committed:
快捷搜索增加祖尔格拉布ZUG

hitechbeijing committed:
快速搜索改为下拉框选择，解决面板位置不够放置所有团本搜索按钮的问题

hitechbeijing committed:
modified:   README.md

hitechbeijing committed:
Update README.md

hitechbeijing committed:
添加屏幕截图

hitechbeijing committed:
Merge branch 'master' of https://github.com/hitechbeijing/meeting-horn

hitechbeijing committed:
Update README.md

hitechbeijing committed:
优化代码

hitechbeijing committed:
Merge branch 'master' of https://github.com/hitechbeijing/meeting-horn

hitechbeijing committed:
修改版本号信息

hitechbeijing committed:
修复reset按钮点击后快速搜索选项无法重置到默认值的问题
暂停版本更新检测（容易误导玩家）

hitechbeijing committed:
修改idleTimer设置逻辑
修改idleTimer文字描述，使其更加通俗易懂

hitechbeijing committed:
20200427更新

hitechbeijing committed:
优化idleTimer设置逻辑

hitechbeijing committed:
添加命令打开/关闭主面板
插件载入后显示欢迎信息

hitechbeijing committed:
modified:   MeetingHorn.toc

hitechbeijing committed:
活动模式增加拍卖
新增繁体中文翻译

hitechbeijing committed:
当创建活动地点在敌对领土时将弹出提示确认

hitechbeijing committed:
修复一个可能引起报错的问题

hitechbeijing committed:
更新readme

hitechbeijing committed:
设置中可以选择是否隐藏小地图图标

hitechbeijing committed:
补充一个文件

hitechbeijing committed:
修复部分简体/繁体中文多语言问题

hitechbeijing committed:
modified:   MeetingHorn.toc

hitechbeijing committed:
优化快速搜索逻辑

hitechbeijing committed:
modified:   MeetingHorn.toc

hitechbeijing committed:
修复部分客户端可能无法载入UI的问题

hitechbeijing committed:
new file:   pkgmeta.yaml

hitechbeijing committed:
解决游戏客户端更新造成依赖库缺失问题

hitechbeijing committed:
重新整合依赖库

hitechbeijing committed:
modified:   README.md

hitechbeijing committed:
modified:   README.md

hitechbeijing committed:
modified:   README.md

hitechbeijing committed:
modified:   README.md

hitechbeijing committed:
modified:   pkgmeta.yaml

hitechbeijing committed:
更换tga图标

hitechbeijing committed:
modified:   pkgmeta.yaml

hitechbeijing committed:
更新简体中文、繁体中文、英文语言

hitechbeijing committed:
活动列表重构测试

hitechbeijing committed:
活动列表重构测试

hitechbeijing committed:
Merge branch 'master' of https://github.com/hitechbeijing/meeting-horn

测试活动列表重构

hitechbeijing committed:
重构快速搜索相关方法

hitechbeijing committed:
解决依赖库问题

hitechbeijing committed:
解决依赖库问题

hitechbeijing committed:
修改api.lua ns.GetPlayerItemLevel方法，修复插件启动时该方法可能报错的问题

hitechbeijing committed:
修复可能无法创建活动的问题

hitechbeijing committed:
Merge branch 'master' of https://github.com/hitechbeijing/meeting-horn

hitechbeijing committed:
deleted:    screenshot.png
